boards/arm/olimexino_stm32/doc/index.rst

```diff
@@ -317,8 +317,10 @@ OLIMEXINO-STM32 board has up to 2 SPIs. The default SPI mapping for Zephyr is:
 I2C
 ===
 
-OLIMEXINO-STM32 board has up to 1 I2C. The default I2C mapping for Zephyr is:
+OLIMEXINO-STM32 board has up to 2 I2C. The default I2C mapping for Zephyr is:
```

Comment:
How about:
```
The OLIMEXINO-STM32 board supports two I2C devices.
```

Reply:
@dbkinder Thanks for the review. Comment addressed.